### ISSUE TITLE
⌨ Accessibility Issue - Profile page - Code bolded text in <strong> tags for screen reader emphasis

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#5004]
## ❌ Current behaviour (issue)
Bolded text on the profile page is currently just coded with css styles. This does not demonstrate to the screenreader that these sections are emphasized.

**WCAG 2.1 Criteria:**

### How to reproduce
1. Environment (prod or UA?) prod
2. Which browser and/or Assistive Technology? Chrome
3. Step-by-step instructions explaining how to reproduce the problem. N/A

## ✔ Expected behaviour (fix)
Encompass the bolded text in `<strong>` tags so that the screen reader knows that there is emphasis on that text. 

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [ ] 
- [ ] 

## Additional information (optional)
1. How does it impact users?:  
Having semantic HTML tags for emphasized text provides contextual information that is currently only available to non AT users.

**Comment by gobyrne** [comment]
So this was actually done deliberately. Eric can deliberate.

**Comment by gobyrne** [comment]
The bolded text in the "My Status" section should be emphasised (move into strong tags). But the rest of the text should probably use something different instead of bolded look. For the most part they are showing key-value pairs. Potentially, we could just remove the font-weight css, but I'll give the designers a chance to weigh in. Hopefully we can find a better way to show key-value pairs.

**Comment by esizer** [comment]
> So this was actually done deliberately. Eric can deliborate.

Sorry, @mnigh  just brought this to my attention. I believe we discussed this but just so it is in writing;

The solution to this is not a simple one and highly subjective. We may need \*__someone__\* to review the copy and determine which text should be verbally emphasized (as if you were reading it aloud) and what text should just stylistically be bold/italic. 

Though, there is some objectivity to this. That is, we should not overuse emphasis (`strong` and `em`) since it has diminishing returns (if everything is emphasized, nothing is). Also, we should avoid bolding entire lines of text, since our `h6` has almost identical font size to the `p` tag, users could confuse an entire line of bold text as a `h6`.

This seems like a good topic for the copy caucus 😬

**Comment by substrae** [comment]
@esizer @mnigh @RM-1978 

So there are a few hard-and-fast rules we should be following here:
- Eric's distinction between what would be emphasized verbally compared to what is bolded for other purposes (whether it be stylistic, visual hierarchy, etc.) is a good place to begin from
- Generally speaking, text that is bolded within a sentence, paragraph, or series of paragraphs should always use the `<strong>` tag, as the text is being bolded for emphasis, not styles. This rule can be approached from the perspective of "if the bolded text exists **within** a different text style, then it's probably bolded for emphasis"
- Bolding for full title text, links, or other standard UI components should **not** use the `<strong> tag, as those are bolded for visual weight only

Happy to chat about ways to make this more robust/easier for everyone.

**Comment by RM-1978** [comment]
I agree with @substrae's thoughts. That said, I do want to drill down on  some recent **cognitive accessibility research** to suggest a go-forward convention for his second point: 

**Research**:  

- Stanford University Research suggests that inconsistent emphasis and overuse of the bold style can make it hard for the user to understand any of your content, let alone what is important for them to retain.  What's more, cognitive accessibility researchers have also found that  **Bold text** should be used to describe key words and phrases (1-3 words).

**Suggested approach** for sentences, paragraphs, or  paragraph series  
I suggest that we follow these parameters to maximize our use of the **Bold** style in sentences or paragraphs: 

- We use the `strong` convention to emphasize strong, clear words that draw user attention to important points or concepts within our sentences.  
- Copy review **bolded sentences** with the editors to see whether the entire sentence should be bolded or whether need to use the style only for  key words or short phrases within the sentence for cognitive simplification.
